src/components/ProjectsGrid.tsx

```diff
@@ -49,7 +49,7 @@ const ProjectsGrid = () => {
         justifySelf={["center", "center", "center", "end"]}
         maxWidth={iconSizes}
       >
-        <Link to="/projects/editorial">
+        <Link to="/projects/fashioneditorial">
```

Comment:
this is supposed to be the constant `FASHION_EDITORIAL_URL` along with all the others in this file